crates/sui-prover/tests/snapshots/prettyprint-enum.move.snap

```diff
@@ -19,11 +18,3 @@ error: prover::ensures assertion does not hold
    │     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    │
    =     at tests/inputs/prettyprint-enum.move:20: foo_spec
-   =         x = ELIDEDu64
-   =     at tests/inputs/prettyprint-enum.move:21: foo_spec
-   =         x = ELIDEDu64
-   =     at tests/inputs/prettyprint-enum.move:12: foo
-   =         result = foo::MyEnum::B{pos0 = ELIDEDu64}
-   =     at tests/inputs/prettyprint-enum.move:17: foo
-   =         res#1#0 = foo::MyEnum::B{pos0 = ELIDEDu64}
```

Comment:
let's add `log::var(res)` to the test to check the pretty-printing of the `enum`, and let's change the code to `MyEnum::B(0)` instead of `MyEnum::B(x)` so the debug message is always the same.